commands/generate_test.go

```diff
@@ -202,6 +202,35 @@ spec:
 		Branch:     "",
 		Version:    "",
 	},
+	{
+		Name: "Read-only root filesystem",
```

Comment:
Is there an equivalent test that shows it's not outputted? We may already have that.

Reply:
If you're OK that it's covered then we can leave it as it is.